Stratis.Bitcoin/Features/BlockStore/BlockStoreLoop.cs

```diff
@@ -69,16 +74,17 @@ public sealed class BlockStoreLoop
         }
 
         /// <summary>
-        /// 
         /// Initialize the BlockStore
-        ///                 
+        /// <para>
         /// If StoreTip is null, the store is out of sync.
+        /// 
```

Comment:
There should be no empty lines in `///` comments, use `<para>` for formatting. You can check that in MSVS when you put your mouse over the name, you will see how it formats and this blank line will be ignored, so in order to make a blank line, you want to use here `</para>` to close your previous paragraph and then start new one with `<para>`